trunk/problems/MAXWELL/ULTRAWEAK_DPG/set_initial_mesh.F90

```diff
@@ -1,114 +1,68 @@
-
 !-------------------------------------------------------------------------------
 !> @brief       Define problem dependent data (multiphysics, BC, approximation)
 !!
 !> @param[out]  Nelem_order  - order of initial mesh elements
 !!
 !> @date        July 2023
 !-------------------------------------------------------------------------------
-   subroutine set_initial_mesh(Nelem_order)
+subroutine set_initial_mesh(Nelem_order)
```

Comment:
Why undo the indentation? I've been trying to fix this in the code (add indentation so there is at least one indent everywhere), it looks bad when we also start each blank line with a comment.

Reply:
Habit, I suppose (when you look at the laser code, you'll see that it doesn't indent the "subroutine" and "end subroutine" parts anywhere) :-) I'm fine indenting those but I wouldn't like default double-indentation (6 spaces) for everything that follows because it takes too much space away from a reasonable line length.

Btw, should we just stop / remove the `!` on blank line?